### PR TITLE
tvOS support

### LINKED ISSUE #369: Add tvOS Cocoapods support
I'd like to use this framework in a tvOS project using Cocoapods but in order to do this you need to explicitly state this project is made for tvOS, too.

For further details see [this blog post](http://blog.cocoapods.org/CocoaPods-0.39/).

### PR DESCRIPTION
**Pull request by Jeehut**
I have finalized the initial tvOS support in my tvOS project within this library, everything works now for me. Please note that this branch was forked by the "feature/arc" branch, so #372 should be merged before this one.

This attempts to fix #369.

### CODE REVIEW

**Comment by adamgit** [comment]
I've cleaned up the build settings, cleaned up the layouts, and made it easier to integrate + maintain your Framework builds.

Unfortunately, this second PR is now giving a merge conflict.

Please merge onto lastest 2.x

**\* NB: I've updated to latest Lumberjack (2.1.0, which fixes tvOS bugs), and improved the Lumberjack integration, along the lines you had here (it's now a proper pre-made framework, etc).

**\* NB: the project is now also ARC across the board. This affects some of your commits I believe.

File: Source/UIKit additions/NSCharacterSet+SVGKExtensions.m [hunk 1]
```diff
@@ -21,7 +21,7 @@ + (NSCharacterSet *)SVGWhitespaceCharacterSet;
     dispatch_once(&onceToken, ^{
 		SVGKitLogVerbose(@"Allocating static NSCharacterSet containing whitespace characters. Should be small, but Apple seems to take up 5+ megabytes each time?");
 		sWhitespaceCharacterSet = [NSCharacterSet characterSetWithCharactersInString:[NSString stringWithFormat:@"%c%c%c%c", 0x20, 0x9, 0xD, 0xA]];
-		[sWhitespaceCharacterSet retain]; // required, this is a non-ARC project.
+		 // required, this is a non-ARC project.
```
Comment (zammbi) [inline Source/UIKit additions/NSCharacterSet+SVGKExtensions.m]: Unneeded comment?

**Review by pavel-chekhau — APPROVED** [review]

**Comment by Jeehut** [comment]
Feel free to fork my fork and update what's needed. I won't have time to update this for quite some time and the project I needed this in is already completed and working. So ...

**Comment by adamgit** [comment]
It's been 2.5 years, and I think all these changes got merged separately / cherry-picked by other commits?

I'm  closing this - but if anyone spots something in it that is still missing from 2.x then please go ahead and create a new PR (I suggest making it small / multiple PR's so they're easier to deal with this time. C.f. previous attempt where I did some cleanup but it was still too much of a change and conflicted with other things going onto the 2.x branch, so I couldn't commit the result even after a couple of us had worked on it :( )

**Pull request closed without merging**